crates/orderbook/src/solver_competition.rs

```diff
@@ -28,6 +28,13 @@ pub trait SolverCompetitionStoring: Send + Sync {
         &self,
         identifier: Identifier,
     ) -> Result<SolverCompetitionAPI, LoadSolverCompetitionError>;
+
+    /// Retrieves the solver competition for the most recent auction.
+    ///
+    /// Returns a `NotFound` error if no solver competition could be found.
+    async fn load_latest_competition(
```

Comment:
Can you document this also in our [Open API](https://github.com/cowprotocol/services/blob/1c63ed3eda8e17865862b55141ac53040fa09aa2/crates/orderbook/openapi.yml#L1) definition to make it available in [swagger](https://api.cow.fi/docs/)?